File: ch02/switch_example.zig
const print = @import("std").debug.print;
pub fn main() !void {
    const value = 3;
    switch (value) {
        // syntax 
        // const expr (case) => <expression>, 
        1 => print("Value is 1", .{}), 
        2 => print("Value is 2", .{}), 
        3 => print("Value is 3", .{}), 
        // default expression 
        else => print("Invalid value", .{})
    }
}

File: ch02/switch_example.c
#include<stdio.h>

int main(){
    int value = 3;
    switch (value) {
    case 1: printf("Value is 1");
        break;
    case 2: printf("Value is 2");
        break;
    case 3: printf("Value is 3");
        break;
    default: printf("Invalid value");
        break;
    }
    return 0;
}